plugins/Jobs.cpp

```diff
@@ -491,11 +513,11 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
             // Are we creating a new job, or updating an existing job?
             if (updateJobID) {
                 // Update the existing job.
-                if(!db.write("UPDATE jobs SET "
-                               "repeat   = " + SQ(SToUpper(job["repeat"])) + ", " +
-                               "data     = JSON_PATCH(data, " + safeData + "), " +
-                               "priority = " + SQ(priority) + " " +
-                             "WHERE jobID = " + SQ(updateJobID) + ";"))
+                if(!db.writeIdempotent("UPDATE jobs SET "
+                                         "repeat   = " + SQ(SToUpper(job["repeat"])) + ", " +
+                                         "data     = JSON_PATCH(data, " + safeOriginalData + "), " +
```

Comment:
Updating a job only happens in this flow with a unique job, right? I just want to make sure

Reply:
Yes, currently we can only update a job if it was `unique` (and we don't currently handle unique jobs in mock requests).